ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,56 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project. It deploys Kubernetes with the Antrea image and tests whether the OVS has hardware offload capabilities or not.
+
+## Workflow
+
+The CI uses three scripts to run the tests:
+
+* start_ci.sh: Build and deploy the Kubernetes test environment including Antrea.
```

Comment:
I took a quick look at the scripts and it seems that the K8s cluster used for testing is a single-Node cluster and that the version used is the latest stable release (as per `https://storage.googleapis.com/kubernetes-release/release/stable.txt`). I think this is worth documenting here.

Reply:
Added a small description of the environment at the start of the section and put the details there.
Please have a look.